adi_doctools/directive/hdl.py

```diff
@@ -4,6 +4,7 @@
 import re
 from os import path, walk
 from os import pardir, makedirs
+from typing import Final
```

Comment:
What is this for?

Reply:
used to test something that was not implemented after all, will be removed